apiserver/facades/agent/storageprovisioner/service.go

```diff
@@ -63,3 +63,10 @@ type StoragePoolGetter interface {
 	// - [storageerrors.PoolNotFoundError] if a pool with the specified name does not exist.
 	GetStoragePoolByName(ctx context.Context, name string) (domainstorage.StoragePool, error)
 }
+
+// ApplicationService is an interface for the application domain service.
+type ApplicationService interface {
+	// WatchApplicationAddRemove returns a watcher that emits application uuids
```

Comment:
Mis-named

Looks like this is my fault though, I'm guessing you copied this from somewhere?

Reply:
Fixed.